services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/RouteOptions.java

```diff
@@ -1195,6 +1215,28 @@ public Builder annotationsList(@Nullable List<String> annotations) {
     @NonNull
     public abstract Builder exclude(@Nullable @DirectionsCriteria.ExcludeCriteria String exclude);
 
+    /**
+     * Include certain road types from routing. The default is to not include anything from the
+     * profile selected. The following include flags are available for each profile:
+     * <p>
+     * {@link DirectionsCriteria#PROFILE_DRIVING}: One of {@link DirectionsCriteria#INCLUDE_HOV2},
+     * {@link DirectionsCriteria#INCLUDE_HOV3}, or {@link DirectionsCriteria#INCLUDE_HOT}.
+     * <p>
+     * {@link DirectionsCriteria#PROFILE_DRIVING_TRAFFIC}: One of
+     * {@link DirectionsCriteria#INCLUDE_HOV2}, {@link DirectionsCriteria#INCLUDE_HOV3}, or
+     * {@link DirectionsCriteria#INCLUDE_HOT}.
+     * <p>
+     * {@link DirectionsCriteria#PROFILE_WALKING}: No includes supported
+     * <p>
+     * {@link DirectionsCriteria#PROFILE_CYCLING}: No includes supported
+     *
+     * @param include a string matching one of the {@link DirectionsCriteria.IncludeCriteria}
+     *                inclusions
+     * @return this builder for chaining options together
+     */
+    @NonNull
+    public abstract Builder include(@Nullable @DirectionsCriteria.IncludeCriteria String include);
```

Comment:
Can you include only one parameter, or can it be a list? If the latter, we should also expose a `List` method variant and test it.

Reply:
Similar to `exclude` https://github.com/mapbox/mapbox-navigation-android/blob/d1218a77e305e04672754835df5fe22a4a68985a/libnavigation-base/src/main/java/com/mapbox/navigation/base/route/RouteExclusions.kt#L15-L21 should we do implement that for `include` from the SDK side for consistency? 🤔

Reply:
Hmm, this should be part of the `mapbox-java` for both include and exclude, same as all the other list paramaters.